src/app.js

```diff
@@ -118,6 +118,8 @@ class SAFEApp extends EventEmitter {
                 let newPath;
                 if (!path || !path.length) {
                   newPath = '/index.html';
+                } else if (path === '/') {
+                  newPath = 'index.html';
```

Comment:
I got you, so we are trying to support the case when the `index.html` is stored at root without a leading `/`. In that case I think we still need to work for fetching `safe://mydomain` (without trailing `/`).